### PR TITLE
adding a section for C# language and Mono framework

### PR DESCRIPTION
**Pull request by tpokorra**

### CODE REVIEW

File: tech/languages/csharp/mono-build-tools-installation.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+```
+$ sudo dnf install nant
+```
+
+# NUnit installation
+
+[NUnit](http://nunit.org/) is very useful for test driven development. To install it, simply type:
+
+```
+$ sudo dnf install nunit
```
Comment (strzibny) [inline tech/languages/csharp/mono-build-tools-installation.md]: it's actually `mono-nunit`

File: tech/languages/csharp/mono-installation.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: Mono
+page: csharp
+order: 1
+section: tech-languages
+description: C# provided by Mono: a Cross-platform, Open Source, .NET development framework
```
Comment (strzibny) [inline tech/languages/csharp/mono-installation.md]: Why should be Cross-platform and Open Source in capitals?

**Comment by strzibny** [comment]
I did some comments inline. Apart from that. Are there some other useful `mono-*` packages?

File: tech/languages/csharp/mono-installation.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+# Mono installation
+
+Fedora comes with [Mono](http://www.mono-project.com/) - a free and open source implementation of the .Net framework. To install it, simply type:
```
Comment (strzibny) [inline tech/languages/csharp/mono-installation.md]: I would like to see `-` being `&ndash;`.

File: tech/languages/csharp/mono-installation.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+```
+$ sudo dnf install mono-devel
+```
+
+This command will install Mono - the runtime environment and associated development tools.
```
Comment (strzibny) [inline tech/languages/csharp/mono-installation.md]: Here as well. I would like to see `-` being `&ndash;`.

File: tech/languages/csharp/mono-build-tools-installation.md (hunk 1 above)
Comment (tpokorra) [inline tech/languages/csharp/mono-build-tools-installation.md]: I was already thinking Fedora 23... 
In F23 the package nunit has obsoleted mono-nunit: https://admin.fedoraproject.org/pkgdb/package/nunit/

I guess I have to make two instructions, one for F21&F22, and one for F23+

File: tech/languages/csharp/mono-installation.md (hunk 2 above)
Comment (tpokorra) [inline tech/languages/csharp/mono-installation.md]: I just copy&pasted from https://admin.fedoraproject.org/pkgdb/package/mono/
I will fix it to lower case

File: tech/languages/csharp/mono-installation.md (hunk 3 above)
Comment (tpokorra) [inline tech/languages/csharp/mono-installation.md]: I looked at https://github.com/developer-portal/content/blob/master/tech/languages/java/java-installation.md and that had a simple dash. I will use two dashes now...

**Comment by tpokorra** [comment]
thanks for your comments. 

I also wrote an email to the Mono SIG this morning, to see if other people come up with some suggestions... https://lists.fedoraproject.org/pipermail/mono/2015-November/000558.html

There are probably other useful mono-* packages. eg. mono-basic with VisualBasic.NET support for mono, but that would probably not fit the csharp section...

**Comment by strzibny** [comment]
Thank you, I will merge it. Feel free to submit a new PR if you have anything to add.

**Comment by strzibny** [comment]
@tpokorra can you also perhaps submit a logotype for Mono to `website` repository?

File: tech/languages/csharp/mono-build-tools-installation.md (hunk 1 above)
Comment (strzibny) [inline tech/languages/csharp/mono-build-tools-installation.md]: I didnt see your inline comments. Now I understand. Can you make changes that reflect this change?

**Pull request merged**